test/integration.test.ts

```diff
@@ -775,21 +775,21 @@ describe('getBranch', () => {
 
     const getBranchOptions = { apiKey: '', apiUrl: '', fetchImpl: {} as FetchImpl };
 
-    process.env = { XATA_BRANCH: branchName };
+    process.env = { NODE_ENV: 'development', XATA_BRANCH: branchName };
```

Comment:
Just curious. Why do we need this env now?

Reply:
We don't for the tests to pass, but ``tsc`` complains that ``NodeEnv`` always needs to be set. Adding it to fix type errors from the compiler